lib/generate.js

```diff
@@ -135,6 +136,40 @@
         "id": "id"
     }
 
+    generate.js = generate.jstestfuzz = function (parser, exportName) {
+        var output = "// Generated automatically by nearley, version " + parser.version + "\n";
```

Comment:
Oh nearley's string creation methodology is so unpleasant 😢

Was there a technical reason that you couldn't use ES6? I didn't realize using ES6 template strings would have such a large code quality impact on Friday.